### PR TITLE
[DEV-279] Make ticket table show more tickets overview

### PR DESCRIPTION
**Pull request by joelmendez1**

### CODE REVIEW

**Comment by joelmendez1** [comment]
image

File: client/src/app-components/page-dropdown.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import DropDown from 'core-components/drop-down';
+
+class PageDropdown extends React.Component {
```
Comment (mredigonda) [inline client/src/app-components/page-dropdown.js:5]: Let's call it `PageSizeDropdown`, no fear of making names a bit longer, specially here where it disambiguates between two meanings.

**Review by mredigonda — COMMENTED** [review]
The functionality is working quite well for the "My Tickets" page, congratulations on that! 😁 The most important part is already completed!

image

In "New Tickets" and "Search Tickets", changing the number of tickets doesn't update the table.

I guess this is because the backend paths weren't updated there, we should either update them, or hide the select from those pages.

Also, let's experiment with having a different design, showing only a number there isn't very descriptive as to what this number does.

If you can make the button smaller, let's try to make it look like:

 this one.

Or at the very least we should add something else to that number:

here

File: client/src/app-components/page-size-dropdown.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        selectedIndex: 1
+    }
+
+    static defaultProps = {
+        showDropDown: true
+    }
+
+    render() {
+        return (
+            this.props.showDropDown ?
```
Comment (mredigonda) [inline client/src/app-components/page-size-dropdown.js:22]: This component should know nothing about this `showDropDown`. The intuition is that you shouldn't try to render a component that is basically an empty div.

Instead, the parent component should conditionally render the entire `PageSizeDropdown` component, depending on that boolean value.

File: client/src/app-components/ticket-query-list.js [hunk 3]
```diff
@@ -65,9 +67,21 @@ class TicketQueryList extends React.Component {
...
 
+    test(selectedIndex) {
+        const currentPath = window.location.pathname;
+        const urlQuery = searchTicketsUtils.getFiltersForURL({
+            filters: {pageSizeIndex: 2},
+            shouldRemoveCustomParam: true,
+            shouldRemoveUseInitialValuesParam: true
+        });
+        urlQuery && history.push(`${currentPath}${urlQuery}`);
+    }
```
Comment (mredigonda) [inline client/src/app-components/ticket-query-list.js:83]: Write a better name for this function based on what it does, it seems you wrote it while developing, is it current? Is it used?

**Review by mredigonda — COMMENTED** [review]
Looks already pretty good, looking forward to merge this one!

I left some minimal comments to take care of, but I already like it a lot!

File: client/src/app-components/ticket-list.js [hunk 4]
```diff
@@ -49,25 +50,28 @@ class TicketList extends React.Component {
     };
```
Comment (mredigonda) [inline client/src/app-components/ticket-list.js:50]: We have a new prop, so let's add its prop type: `showDropDown` (change its name too, see other comments).

**Review by mredigonda — APPROVED** [review]
LGTM 🎉.

**Pull request merged**